src/main/java/nextstep/version/refactor/domain/Direction.java

```diff
@@ -0,0 +1,7 @@
+package nextstep.version.refactor.domain;
+
+public enum Direction {
+    PASS,
+    NEXT,
+    BEFORE;
```

Comment:
```suggestion
    PASS(0),
    NEXT(1),
    BEFORE(-1);
```

위와 같이 Direction이 이동할 때의 값을 가지고 있으면 좋은 점이 있지 않을까?

Reply:
해당 객체에서 이동 거리에 대한 반환 메소드가 있으면 더 좋을 것 같아서 
추가적으로 메소드를 구현하여 반환하도록 수정했습니다!